tests/phpunit/Checker/AJAX_Runner_Tests.php

```diff
@@ -10,9 +10,16 @@
 use WordPress\Plugin_Check\Test_Data\Empty_Check;
 use WordPress\Plugin_Check\Test_Data\Error_Check;
 use WordPress\Plugin_Check\Test_Data\Runtime_Check;
+use WordPress\Plugin_Check\Test_Utils\Traits\With_Mock_Filesystem;
 
 class AJAX_Runner_Tests extends WP_UnitTestCase {
 
+	use With_Mock_Filesystem;
+
+	public function set_up() {
+		$this->set_up_mock_filesystem();
+	}
```

Comment:
What is the rationale for setting that up here unrelated to any actual test changes in this class? Please clarify, but also add a code comment so that it remains clear for people that look at this code in the future.

Reply:
@felixarntz the changes in the PR now mean we use the file system to determine if we can return runtime checks. After the update we were experiencing errors in some of the tests in the [GitHub actions](https://github.com/10up/plugin-check/actions/runs/5058336367/jobs/9078334371).

This uses the mock file system in those tests when the runner is prepared and fixes the issues in the GitHub actions.